main.go

```diff
@@ -184,6 +192,37 @@ func main() {
 		return
 	}
 
+	if err := setupReconcilers(mgr, deploymentsNamespace, enableMetrics, enableTracing, alwaysAcceptAdmissionReviewsOnDeploymentsNamespace); err != nil {
+		retcode = 1
+		return
+	}
+
+	if err := setupWebhooks(mgr, deploymentsNamespace); err != nil {
+		retcode = 1
+		return
+	}
+
+	//+kubebuilder:scaffold:builder
```

Comment:
question: this is for injecting the kubebuilder scaffolded code for the controllerManager that does the `setupWithManager()`, correct?
I couldn't find any docs, only this [comment](https://togithub.com/kubernetes-sigs/kubebuilder/issues/1487#issuecomment-1485346413). The relevant code seems to be [pkg/machinery](https://github.com/kubernetes-sigs/kubebuilder/blob/a897dae204d9b32483d14c60a78573fca6827378/pkg/machinery).